ros_config:
  init_pose_topic: ~init_pose


agent_config:
  action:
    h_rotation: 45.0
  agent_class: SloopMosTopo2DAgent
  belief:
    prior:
      G: uniform
  foref_models_dir: /home/kaiyuzh/repo/robotdev/shared/ros/sloop_ros/tests/sloop_object_search/../../models
  no_look: true
  object_symbol_map:
    GreenToyota: G
  objects:
    G:
      class: car
      color:
      - 100
      - 200
      - 80
      transition:
        class: sloop_object_search.oopomdp.StaticObjectTransitionModel
  robot:
    detectors:
      G:
        class: sloop_object_search.oopomdp.FanModelSimpleFP
        params:
        - fov: 90
          max_range: 15
          min_range: 0
        - - 0.9
          - 0.1
          - 0.25
    id: robot0
  spacy_model: en_core_web_lg
  targets:
  - G
  topo_map_args:
    degree:
    - 3
    - 5
    node_coverage_radius: 3.0
    num_place_samples: 10
    resample_prob_thres: 0.25
    seed: 1120
    sep: 4.0
planner_config:
  local_search:
    planner: pomdp_py.POUCT
    planner_args:
      exploration_const: 1000
      max_depth: 10
      planning_time: 0.15
  planner: sloop_object_search.oopomdp.planner.hier2d.HierarchicalPlanner
  subgoal_level:
    exploration_const: 1000
    max_depth: 20
    planning_time: 0.25
task_config:
  map_name: austin
  max_steps: 100
  visualizer: sloop_object_search.oopomdp.agent.VizSloopMosTopo
  viz_params:
    res: 20
